rails/spec/requests/api/public_story_spec.rb

```diff
@@ -0,0 +1,71 @@
+require "rails_helper"
+
+RSpec.describe "Public Story Detail Endpoint", type: :request do
+  let!(:community) { FactoryBot.create(:community, public: true, name: "Cool Community") }
+  let!(:place_1) { create(:place, community: community, region: "the internet") }
+  let!(:place_2) { create(:place, community: community, type_of_place: "online") }
+  let!(:speaker_1) { create(:speaker, community: community) }
+  let!(:speaker_2) { create(:speaker, community: community, speaker_community: "ruby for good") }
+
+  let!(:story) do
+    create(
+      :story,
+      id: 123,
+      community: community,
+      topic: "Testing",
+      language: "English",
+      places: [place_1, place_2],
+      speakers: [speaker_1, speaker_2],
+      permission_level: :anonymous
+    )
+  end
+
+  def json_response
+    JSON.parse(response.body)
+  end
+
+  it "returns 404 when community can't be found" do
```

Comment:
should this be "when story can't be found"?

Reply:
No. It's testing that a valid place route that includes a community slug that can't be found, it correctly returns not found.